### DISCUSSION TITLE
Active/inactive repositories

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1807] [General]
We have repositories in our organisation that receive differing amounts of maintenance. Should we make this explicit in some way?

Are we maintaining bip47? (https://github.com/rust-bitcoin/rust-bip47)

### Inactive 

Could these potentially be archived? 

- https://github.com/rust-bitcoin/rust-wallet
- https://github.com/rust-bitcoin/constants

### Org repo pinning

@apoelstra could you please change the 6 pinned repos to be 6 active ones? Possibly

- https://github.com/rust-bitcoin/rust-miniscript
- https://github.com/rust-bitcoin/rust-bitcoin
- https://github.com/rust-bitcoin/rust-secp256k1
- https://github.com/rust-bitcoin/rust-bech32
- https://github.com/rust-bitcoin/rust-bitcoincore-rpc
- https://github.com/rust-bitcoin/www.rust-bitcoin.org

**Comment by apoelstra** [comment]
Yeah, `constants` I'm happy to archive. We may yet resurrect this crate but I think if we do, we'll do it inside the rust-bitcoin repo.

As for rust-wallet or rust-bip47, I am not a maintainer of either of those (though I probably did sign off on moving them into the org, many years ago), and don't want to speak for their maintainers as to whether they should be archived or what.

**Comment by stevenroose** [comment]
As for rust-wallet, Tamas at the time "handed" it to me to maintain and at the time I had some ideas to turn it into something that kinds BDK turned into at the moment. i mean BDK does a lot more, but I currently don't really have the cycles/interest to work on what I initially intended to do with rust-wallet. I think it'd be fair to archive it. I don't think anyone is using it anymore and it's kinda an amalgam of different things. Feel free to archive it.

It would be a nice place to do things like address generation from a bip32 seed using all the different bip44-and-alike schemes and from descriptors. Facilitate utxo selection etc. 

Like all of us, I've been pondering with writing a wallet for a while and maybe if I ever get to it, I could try to generalize some aspects into rust-wallet, but that's just dreaming :)

**Reply by apoelstra** [reply]
Would it be reasonable to point people to bdk in the README before archiving?

**Reply by stevenroose** [reply]
Could be in the GitHub description so no one needs to push to the readme, right? Or do you think that's insufficient?

**Reply by apoelstra** [reply]
I think it should be in the README. I rarely look at the github description for a repo.

**Reply by tcharding** [reply]
Done: https://github.com/rust-bitcoin/rust-wallet/pull/37
